Revit_Core_Engine/Query/LocationPoint.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Autodesk.Revit.DB;
+using BH.oM.Base.Attributes;
+using System.ComponentModel;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Returns the 3D location point of a given Revit element. The method determines the most appropriate point based on the element type, such as the spatial element calculation point, the element's location point, the midpoint of its location curve, or the center of its bounding box.")]
+        [Input("element", "The Revit element from which to extract the location point.")]
+        [Input("useRoomCalculationPoint", "If true and the element is a FamilyInstance with a spatial element calculation point, use that point as the location.")]
+        [Output("locationPoint", "The 3D point representing the location of the input Revit element")]
+        public static XYZ LocationPoint(this Element element, bool useRoomCalculationPoint = false)
+        {
```

Comment:
Please add a null check, otherwise the method will crash with null reference exception most likely

Reply:
Added 👍